instance/models/openedx_appserver.py

```diff
@@ -176,6 +176,10 @@ class Meta:
             "behind a firewall. (In the django admin, separate group names with a comma.)"
         )
     )
+    enable_vm_dns_records = models.BooleanField(
+        default=False,
+        help_text="Set additional DNS records pointing directly to the active app servers."
+    )
```

Comment:
@smarnach The reason I ask is that when there is an issue with an instance, it would gain time to be able to directly SSH a predictible domain name, rather than having to go through the Instance manager to find the right appserver and its IP. When I was trying to figure out why Cloudera was down, on my phone in the bus at 5am last week-end, I would have really appreciated it : )